### PR TITLE
oleksijt_task14_print_sentense_in_reverse_order

### PR DESCRIPTION
**Pull request by Oleksijt**
Please review.

### CODE REVIEW

File: oleksijt/oleksijt_task14_print_sentense_in_reverse_order.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+Input example:
+This is a test sentense
+
+Output example:
+esnetnes tset a si sihT
+"""
+
+
+def print_reverse(sentence):
+    print(sentence[::-1])
```
Comment (pivancy) [inline oleksijt/oleksijt_task14_print_sentense_in_reverse_order.py]: Please use `return` instead.

Comment (Oleksijt) [reply oleksijt/oleksijt_task14_print_sentense_in_reverse_order.py]: Fixed. I did so becouse the task was "...function which prints a sentense...".

Comment (pivancy) [reply oleksijt/oleksijt_task14_print_sentense_in_reverse_order.py]: You should print the result of function :)
In most cases function should return something.

**Comment by pivancy** [comment]
:+1:

**Pull request merged**